src/layouts/tv/home/views.livetv.html

```diff
@@ -7,7 +7,7 @@
                     <div class="cardPadder-square"></div>
                     <div class="cardContent cardContent-shadow">
                         <div class="cardImageContainer coveredImage">
-                            <i class="cardImageIcon material-icons">menu</i>
+                            <em class="cardImageIcon material-icons">menu</em>
```

Comment:
Don't use `<em>` for icons, use `<span>`.

Sonarqube's parser doesn't know about icons, so they're recommending the wrong element for this case.

The change is meant to better support screen readers, which treat emphasis differently than normal text. What we want to do is inform these screen readers that the element is "a bunch of content" that isn't really anything specific.

Then we describe it more through `aria-label` and `aria-hidden`. The first one is used in cases where the icon represents an action to be performed (for example an "Add" button). The second one is to be used when an element is decorative (So not necessary for the comprehension of the screen).